container/sagemaker/nginx.conf.template

```diff
@@ -51,6 +51,10 @@ http {
         %FORWARD_INVOCATION_REQUESTS%;
     }
 
+    location ~ ^/models/(.*)/invoke {
+        js_content invocations;
```

Comment:
what if they do have custom python handlers? or is that not allowed?

Reply:
discussed offline -- this feature will be added separately

Reply:
As we discussed offline. We will first support dynamic endpoint invocation url without python pre/post-processing.